config/getdate.sh

```diff
@@ -7,4 +7,9 @@
 # Provide a way to override build date for reproducible build results
 # See https://reproducible-builds.org/ for why this is good.
 
-date --date="@${SOURCE_DATE_EPOCH:-$(date +%s)}" "$@"
+if test -n "`uname -a | grep -i darwin`"; then
+    # MacOS's "date" command is different than GNU date.
+    date -j -f '%s' "${SOURCE_DATE_EPOCH:-$(date +%s)}" "$@"
```

Comment:
https://reproducible-builds.org/docs/source-date-epoch/ has this variant:
```

SOURCE_DATE_EPOCH="${SOURCE_DATE_EPOCH:-$(date +%s)}"
BUILD_DATE=$(date -u -d "@$SOURCE_DATE_EPOCH" "$DATE_FMT" 2>/dev/null || date -u -r "$SOURCE_DATE_EPOCH" "$DATE_FMT" 2>/dev/null || date -u "$DATE_FMT")
```
and the middle part with `-r` should handle FreeBSD+MacOS date.

I would adapt it here as
```
SOURCE_DATE_EPOCH="${SOURCE_DATE_EPOCH:-$(date +%s)}"
date -u -d "@$SOURCE_DATE_EPOCH" "$@" 2>/dev/null || date -u -r "$SOURCE_DATE_EPOCH" "$@" 2>/dev/null || date -u "$@"
```